packages/analytics/analytics-metric-provider/vite.config.ts

```diff
@@ -0,0 +1,29 @@
+import sharedViteConfig, { sanitizePackageName } from '../../../vite.config.shared'
+import { resolve } from 'path'
+import { defineConfig, mergeConfig } from 'vite'
+
+// Package name MUST always match the kebab-case package name inside the component's package.json file and the name of your `/packages/{package-name}` directory
+const packageName = 'analytics-metric-provider'
+const sanitizedPackageName = sanitizePackageName(packageName)
+
+// Merge the shared Vite config with the local one defined below
+const config = mergeConfig(sharedViteConfig, defineConfig({
+  build: {
+    lib: {
+      // The kebab-case name of the exposed global variable. MUST be in the format `kong-ui-public-{package-name}`
+      // Example: name: 'kong-ui-public-demo-component'
+      name: `kong-ui-public-${sanitizedPackageName}`,
+      entry: resolve(__dirname, './src/index.ts'),
+      fileName: (format) => `${sanitizedPackageName}.${format}.js`,
+    },
+  },
```

Comment:
After moving `i18n` package to peerDependencies, you should externalize the dependency
```suggestion
  build: {
    lib: {
      // The kebab-case name of the exposed global variable. MUST be in the format `kong-ui-public-{package-name}`
      // Example: name: 'kong-ui-public-demo-component'
      name: `kong-ui-public-${sanitizedPackageName}`,
      entry: resolve(__dirname, './src/index.ts'),
      fileName: (format) => `${sanitizedPackageName}.${format}.js`,
    },
    rollupOptions: {
      // Make sure to externalize deps that shouldn't be bundled into your library
      external: ['@kong-ui-public/i18n'],
      output: {
        // Provide global variables to use in the UMD build for externalized deps
        globals: {
          '@kong-ui-public/i18n': 'kong-ui-public-i18n',
        },
      },
    },
  },
```

Reply:
Done.